src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperScopeManager.cs

```diff
@@ -31,18 +32,36 @@ public TagHelperScopeManager()
         /// <param name="executeChildContentAsync">A delegate used to execute the child content asynchronously.</param>
         /// <param name="startWritingScope">A delegate used to start a writing scope in a Razor page.</param>
         /// <param name="endWritingScope">A delegate used to end a writing scope in a Razor page.</param>
+        /// <param name="parentExecutionContext">The parent <see cref="TagHelperExecutionContext"/>.</param>
```

Comment:
hmm, sources are normally the first parameter in copy constructors and start-sub-scope methods.  probably good for `parentExecutionContext` to be first here.